bliss/models/prior.py

```diff
@@ -74,18 +74,18 @@ class ImagePrior(pl.LightningModule):
 
     def __init__(
         self,
-        n_bands: int = 1,
-        slen: int = 50,
-        tile_slen: int = 2,
-        min_sources: int = 0,
-        max_sources: int = 2,
-        mean_sources: int = 0.4,
+        n_bands: int,
+        slen: int,
+        tile_slen: int,
+        min_sources: int,
+        max_sources: int,
+        mean_sources: int,
+        f_min: float,
+        f_max: float,
+        alpha: float,
+        prob_galaxy: float,
         loc_min: float = 0.0,
```

Comment:
Some of the tests (at least, the M2 test) currently depend on certain arguments having default values, such as `loc_min, loc_max`. I can't change these without altering the tests, which I didn't want to get into for this issue.

Reply:
I believe you just need to modify `tests/m2/m2.yaml`, which should be okay.

Reply:
I'll suggest the change on here

Reply:
Actually, I don't think these options are even necessary; we always have loc_min=0 and loc_max=1